application/F3DStarter.cxx

```diff
@@ -956,6 +960,44 @@ int F3DStarter::Start(int argc, char** argv)
       }
     }
 
+    // Process Command Script file
+    const std::string& commandScriptFile = 
+      this->Internals->AppOptions.CommandScriptFile;
+    if (!commandScriptFile.empty())
+    {
+      std::ifstream scriptFile(commandScriptFile);
+      if (scriptFile.is_open())
+      {
+        std::string command;
+        while (std::getline(scriptFile, command))
+        {
+          if (!command.empty() && command[0] != '#')
+          {
+            try
+            {
+              interactor.triggerCommand(command);
+            }
+            catch (const std::exception& e)
+            {
+              f3d::log::error("Exception caught while processing command: ", e.what());
+              return EXIT_FAILURE;
+            }
+            catch (...)
+            {
+              f3d::log::error("Unknown exception caught while processing command.");
+              return EXIT_FAILURE;
+            }
```

Comment:
Nevermind, reading the doc I wrote:

```
   * Throw an interactor::command_runtime_exception if the command callback
   * throw an unrecognized exception. Note that default commands cannot throw such
   * an exception.
   */
  virtual bool triggerCommand(std::string_view command) = 0;

```

Since in F3D application, we only have access to default commands, there is no need to catch anything. Sorry about that.